sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeCustomEntitiesTests.cs

```diff
@@ -46,7 +46,7 @@ public RecognizeCustomEntitiesTests(bool isAsync, TextAnalyticsClientOptions.Ser
             "petróleo",
         };
 
-        private static readonly List<string> e_batchConvenienceDocuments = new List<string>
+        private static readonly List<string> s_englishBatchConvenienceDocuments = new List<string>
```

Comment:
is "e_" stands for 'English' and "s_" prefix stands for 'Spanish'?

Reply:
I think that was the original intention, but "e_" is not a valid prefix per C# standards. On the other hand, "s_" is used to prefix static classes.